### DISCUSSION TITLE
Option to adjust the Offset of the Sprites Directly/Individually

### DISCUSSION THREAD

**Discussion by AxZelAnims** [godotengine/godot-proposals#8368] [Editor]
AnimatedSprite's Offset Option is useful in its own way. But some Sprites have different sizes and needed their offset to be adjusted individually. 

E.g. 
Default Example Offset (based on Idle Animation)
X Offset 0.5
Y Offset 0.3
Running Animation Sprites will use "Default Example Offset" but has a different size and will have different position. It needs to have its own X and Y Offset.

I tried setting the required Offset to every first frame of the animation in AnimationPlayer and it works but it will sometimes break and doing it over and over to a bunch of Animations may prove a problem. 

Another workaround is to manually set the required Offset Position via script when that Animation is playing. (Haven't tested yet)

But I suggest that we have an option to manually modify each Sprite's Offset Position. Most possible option would be in the Import Settings.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/665 and https://github.com/godotengine/godot-proposals/issues/2060.
